src/ucp/core/ucp_request.h

```diff
@@ -52,6 +53,17 @@ enum {
         } \
     }
 
+/*
+ * - Mark the request as completed
+ * - Otherwise - the request might be released - if so, return it to mpool.
+ */
+#define ucp_request_rma_complete(_req) \
```

Comment:
why do we need another macro here? if needed we can add more debug info to the macro params of to the request itself